VideoWeb/VideoWeb.EventHub/Handlers/Core/EventHandlerBase.cs

```diff
@@ -77,7 +77,15 @@ await HubContext.Clients.Group(Hub.EventHub.VhOfficersGroupName)
         /// <returns></returns>
         protected async Task PublishConferenceStatusMessage(ConferenceStatus hearingEventStatus)
         {
-            SourceConference.UpdateConferenceStatus(hearingEventStatus);
+            if (hearingEventStatus == ConferenceStatus.Closed)
+            {
+                // Fetch the conference to also get the updated closed date time from video api
+                SourceConference = await conferenceService.ForceGetConference(SourceConference.Id);
```

Comment:
It might not be worth doing a hard refresh here. The difference would be marginal so we could just set the close time to now on the conference model